src/models_test.ts

```diff
@@ -546,7 +549,11 @@ describeMathCPU('loadModel from URL', () => {
       (fileBufferMap:
            {[filename: string]: Float32Array|Int32Array|ArrayBuffer}) => {
         spyOn(window, 'fetch').and.callFake((path: string) => {
-          return new Response(fileBufferMap[path]);
+          return new Promise((res, rej) => {
```

Comment:
Google style prefers explicit, unambiguous names to short, succinct ones. So use `resolve` and `reject` instead.

Reply:
Thanks - I've updated the names.